Gemfile

```diff
@@ -141,3 +141,5 @@ gem 'active_model_serializers', github: 'rails-api/active_model_serializers', re
 gem 'aws-sdk', '~> 2'
 
 gem 'rubocop', require: false, group: [:development, :staging]
+# Reduce volume of production logs
+gem 'lograge'
```

Comment:
if this is used only in production, is it worth putting it in a production Gemfile group?